### DISCUSSION TITLE
Unable to run in fargate containers

### DISCUSSION THREAD

**Discussion by SimonMarchantGB** [artilleryio/artillery#3383] [Artillery]
I've encountered an issue when running artillery in a fargate container whereby it is failing to pull it's image (below)

> CannotPullContainerError: pull image manifest has been retried 1 time(s): failed to resolve ref 301676560329.dkr.ecr.eu-west-1.amazonaws.com/artillery-pro/aws-ecs-node:v2-f4fd35b6295e55952cab43021c8223dab86666f1: 301676560329.dkr.ecr.eu-west-1.amazonaws.com/artillery-pro/aws-ecs-node:v2-f4fd35b6295e55952cab43021c8223dab86666f1: not found

Does the free version no longer support fargate running or has anyone else encountered this issue and worked around it?

**Comment by hassy** [comment]
Hi @SimonMarchantGB - this looks like you're trying to runn a test with Artillery Pro. It's been deprecated back in June and is no longer supported. You can use the `run-fargate` command instead now, guide here: https://www.artillery.io/docs/load-testing-at-scale/aws-fargate

**Reply by LiamJBerridge** [reply]
Hey @hassy I've also had this issue whilst using the run-fargate command. I assumed it was deprecated and there was no longer any free support for fargate with artillery?

**Reply by SimonMarchantGB** [reply]
Some further info on the artillery version and run process I'm following

VERSION INFO:

Artillery: 2.0.0-38
Node.js:   v18.18.2
OS:        darwin

artillery run-fargate--region eu-west-1 --count 1  --config config.yml scenarios/combinedFlow.yml --dotenv "../../.env" 

In the aws account there is an artilleryio-cluster in place with no resources attached. (should this be deleted)

**Reply by hassy** [reply]
`run-fargate` should work - can you share the error you're seeing @LiamJBerridge?

@SimonMarchantGB can you upgrade to the most recent version of Artillery please (v2.0.20) and follow the setup guide in the doc I linked? What happens when you run that `run-fargate` command exactly? (The original error you shared is an Artillery Pro error which is no longer supported.)

**Reply by LiamJBerridge** [reply]
Hey @hassy its the same error as @SimonMarchantGB 

CannotPullContainerError: pull image manifest has been retried 1 time(s): failed to resolve ref 301676560329.dkr.ecr.eu-west-1.amazonaws.com/artillery-pro/aws-ecs-node:v2-f4fd35b6295e55952cab43021c8223dab86666f1: 301676560329.dkr.ecr.eu-west-1.amazonaws.com/artillery-pro/aws-ecs-node:v2-f4fd35b6295e55952cab43021c8223dab86666f1: not found

**Reply by hassy** [reply]
@LiamJBerridge that's an [Artillery Pro](https://www.artillery.io/docs/artillery-pro/installing-artillery-pro) error. Artillery Pro was the original add-on that added Fargate support to Artillery. It was deprecated back in June this year and is no longer supported. You need to upgrade to the most recent release of Artillery and setup & use the `run-fargate` command instead.

**Reply by LiamJBerridge** [reply]
Ok, Thanks @hassy! Much appreciated, I'll try that 👍

**Reply by SimonMarchantGB** [reply]
I can confirm @hassy  that this has resolved the error. Happy for the subject to be closed.
